packages/api-explorer/src/scenes/HomeScene/HomeScene.tsx

```diff
@@ -13,20 +39,21 @@ interface DocHomeParams {
   specKey: string
 }
 
+export const StyledHeading = styled(Heading)``
+
+StyledHeading.defaultProps = {
+  as: 'h1',
+  fontSize: 'xxxlarge',
+  fontWeight: 'semiBold',
+  mb: 'xsmall',
+}
```

Comment:
Seems like this should leverage the `StyledHeading` component defined elsewhere.

Reply:
Also, since it's all just props it should be a `styled` call - just prop assignment to `Heading`

Reply:
Updated.